log_entry.go

```diff
@@ -35,6 +35,7 @@ type CommonLogEntryField struct {
 	Contexts               []Context         `json:"contexts,omitempty"`
 	AcknowledgementTimeout int               `json:"acknowledgement_timeout"`
 	EventDetails           map[string]string `json:"event_details,omitempty"`
+	Assignees              []APIObject       `json:"assignees,omitempty"`
```

Comment:
@tautvydass I've seen @stmcallister make earlier comments about not wanting to accept changes to this library that rely on undocumented API features. Looking through the API docs I don't see this field documented, and so I worry that this might not be accepted.

@stmcallister would you be able to confirm that?

Reply:
In hunting around, I've discovered this is [obscurely documented](https://developer.pagerduty.com/api-reference/c2NoOjI3NDc5OTk-assign-log-entry). I'll take it.